installing/installing_sno/install-sno-preparing-to-install-sno.adoc

```diff
@@ -0,0 +1,16 @@
+[id="preparing-to-install-sno"]
+= Preparing to install on a single node
```

Comment:
I have made this change. I do not understand why github behaves this way. However, it is possible to make a change, stage it for commit, make another change, and stage that change for commit too. I suspect this has something to do with squashing commits. However, I have made the change and it is building properly on my local copy.

Reply:
@mikemckiernan @vikram-redhat